### PR TITLE
faq: add some faqs to documention

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
faq: add some faqs to documention

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/10061
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: faq/manage-cluster-faq.md [hunk 1]
```diff
@@ -69,6 +69,14 @@ TiDB 目前支持 [`wait_timeout`](/system-variables.md#wait_timeout) 和 [`inte
 
 关于 TiDB 版本的管理策略，可以参考 [TiDB 版本规则](/releases/versioning.md)。
 
+### 部署和维护 TiDB 集群的运营成本如何？
+
+TiDB 提供了一些特性和[工具](/ecosystem-tool-user-guide.md)，可以帮助你以低成本管理集群：
+
+- 在运维方面，[TiUP](/tiup/tiup-documentation-guide.md) 作为包管理器，简化了部署、扩缩容、升级和其他运维任务。
+- 在监控方面，[TiDB 监控框架](/tidb-monitoring-framework.md) 使用 [Prometheus](https://prometheus.io/) 存储监控和性能指标，使用 [Grafana](https://grafana.com/grafana/) 可视化这些指标。Grafana 内置了数十个面板，覆盖了数百个指标。
+- 在故障诊断方面，[TiDB 集群问题导图](/tidb-troubleshooting-map.md) 汇总了 TiDB 服务器和其他组件的常见问题。你可以使用这个导图来诊断和解决遇到的相关问题。
```
Comment (TomShawn) [inline faq/manage-cluster-faq.md:78]:
```suggestion
- 在故障诊断方面，[TiDB 集群问题导图](/tidb-troubleshooting-map.md)汇总了 TiDB 服务器和其他组件的常见问题。你可以使用这个导图来诊断和解决遇到的相关问题。
```

File: faq/manage-cluster-faq.md [hunk 2]
```diff
@@ -256,6 +271,12 @@ GROUP BY db_name , table_name;
 
 本小节介绍 TiKV 管理中的常见问题、原因及解决方法。
 
+### 如何为合规性或多租户应用程序指定数据位置？
+
+可以使用 [放置规则 (Placement Rules)](/placement-rules-in-sql.md) 为合规性或多租户应用程序指定数据位置。
```
Comment (TomShawn) [inline faq/manage-cluster-faq.md:276]:
```suggestion
可以使用[放置规则 (Placement Rules)](/placement-rules-in-sql.md) 为合规性或多租户应用程序指定数据位置。
```

File: faq/high-reliability-faq.md [hunk 3]
```diff
@@ -8,6 +8,10 @@ aliases: ['/docs-cn/dev/faq/high-reliability-faq/']
 
 本文档介绍高可靠相关的常见问题。
 
+## TiDB 是否支持数据加密？
+
+支持。要加密传输中的数据，可以[在 TiDB 客户端和服务器之间启用 TLS](/enable-tls-between-clients-and-servers.md)。要加密存储引擎中的数据，可以启用[透明数据加密（TDE）](/encryption-at-rest.md)。
```
Comment (shichun-0415) [inline faq/high-reliability-faq.md:13]:
```suggestion
支持。要加密传输中的数据，可以[在 TiDB 客户端和服务器之间启用 TLS](/enable-tls-between-clients-and-servers.md)。要加密存储引擎中的数据，可以启用[透明数据加密 (TDE)](/encryption-at-rest.md)。

```

File: faq/manage-cluster-faq.md [hunk 4]
```diff
@@ -69,6 +69,14 @@ TiDB 目前支持 [`wait_timeout`](/system-variables.md#wait_timeout) 和 [`inte
 
 关于 TiDB 版本的管理策略，可以参考 [TiDB 版本规则](/releases/versioning.md)。
 
+### 部署和维护 TiDB 集群的运营成本如何？
+
+TiDB 提供了一些特性和[工具](/ecosystem-tool-user-guide.md)，可以帮助你以低成本管理集群：
+
+- 在运维方面，[TiUP](/tiup/tiup-documentation-guide.md) 作为包管理器，简化了部署、扩缩容、升级和其他运维任务。
+- 在监控方面，[TiDB 监控框架](/tidb-monitoring-framework.md) 使用 [Prometheus](https://prometheus.io/) 存储监控和性能指标，使用 [Grafana](https://grafana.com/grafana/) 可视化这些指标。Grafana 内置了数十个面板，覆盖了数百个指标。
```
Comment (shichun-0415) [inline faq/manage-cluster-faq.md:77]:
```suggestion
- 在监控方面，[TiDB 监控框架](/tidb-monitoring-framework.md)使用 [Prometheus](https://prometheus.io/) 存储监控和性能指标，使用 [Grafana](https://grafana.com/grafana/) 可视化这些指标。Grafana 内置了数十个面板，覆盖了数百个指标。
```

File: faq/manage-cluster-faq.md [hunk 5]
```diff
@@ -386,8 +407,12 @@ TiDB 设计的目标就是针对 MySQL 单台容量限制而被迫做的分库
...
 ### TiDB 主要备份方式？
 
-目前，数据量大时（大于 1 TB）推荐使用 [BR](/br/backup-and-restore-overview.md) 进行备份。其他场景推荐使用 [Dumpling](/dumpling-overview.md) 进行备份。
+目前，数据量大时（大于 1 TB）推荐使用 [Backup & Restore (BR)](/br/backup-and-restore-overview.md) 进行备份。其他场景推荐使用 [Dumpling](/dumpling-overview.md) 进行备份。
 
 尽管 TiDB 也支持使用 MySQL 官方工具 `mysqldump` 进行数据备份和恢复，但其性能低于 [Dumpling](/dumpling-overview.md)，并且 `mysqldump` 备份和恢复大量数据的耗费更长。
 
 其他备份恢复相关问题，可以参考[备份与恢复常见问题](/faq/backup-and-restore-faq.md)。
+
+### 备份和恢复的速度如何？
```
Comment (shichun-0415) [inline faq/manage-cluster-faq.md:416]: 感觉 line 408 到 line 418 都更适合放在[备份恢复 FAQ](/faq/backup-and-restore-faq)。Tom 觉得呢？如果同意，等这个 PR 合并后，我一起提 PR 整合一下吧。

File: faq/sql-faq.md [hunk 6]
```diff
@@ -8,6 +8,29 @@ aliases: ['/docs-cn/dev/faq/sql-faq/']
...
+
+支持。你可以在具有唯一[二级索引](/develop/dev-guide-create-secondary-indexes.md)的非主键列上设置 [`NOT NULL` 约束](/constraints.md#非空约束)。在这种情况下，该列用作二级键。
+
+## TiDB 在对大表执行 DDL 操作时，性能表现如何？
+
+TiDB 在对大表执行 DDL 操作时，一般不会有什么问题。TiDB 支持在线 DDL 操作，且这些 DDL 操作不会阻塞 DML 操作。
+
+对于添加列、删除列或删除索引等 DDL 操作，TiDB 可以快速完成这些操作。
+
+对于添加索引等重量级 DDL 操作，TiDB 需要进行回填 (backfill) 操作，这个过程需要较长的时间（取决于表的大小）和额外的资源消耗。对在线业务的影响可调节。TiDB 可以通过多线程进行 backfill，资源消耗可通过以下系统变量进行设置：
```
Comment (shichun-0415) [inline faq/sql-faq.md:21]: “重量级 DDL 操作”读起来感觉略微奇怪，可以直接删掉重量级么？或者改成更具体的表述。

File: faq/tidb-faq.md [hunk 7]
```diff
@@ -81,14 +81,30 @@ Usage of ./bin/tidb-server:
...
+
+### 1.1.13 TiDB 是否支持 XA？
 
 虽然 TiDB 的 JDBC 驱动用的就是 MySQL JDBC (Connector/J)，但是当使用 Atomikos 的时候，数据源要配置成类似这样的配置：`type="com.mysql.jdbc.jdbc2.optional.MysqlXADataSource"`。MySQL JDBC XADataSource 连接 TiDB 的模式目前是不支持的。MySQL JDBC 中配置好的 XADataSource 模式，只对 MySQL 数据库起作用（DML 去修改 redo 等）。
 
 Atomikos 配好两个数据源后，JDBC 驱动都要设置成 XA 模式，然后 Atomikos 在操作 TM 和 RM (DB) 的时候，会通过数据源的配置，发起带有 XA 指令到 JDBC 层。JDBC 层 XA 模式启用的情况下，会对 InnoDB（如果是 MySQL 的话）下发操作一连串 XA 逻辑的动作，包括 DML 去变更 redo log 等，就是两阶段递交的那些操作。TiDB 目前的引擎版本中，没有对上层应用层 JTA/XA 的支持，不解析这些 Atomikos 发过来的 XA 类型的操作。
 
 MySQL 是单机数据库，只能通过 XA 来满足跨数据库事务，而 TiDB 本身就通过 Google 的 Percolator 事务模型支持分布式事务，性能稳定性比 XA 要高出很多，所以不会也不需要支持 XA。
 
+### 1.1.14 TiDB 如何在不影响性能的情况下支持对列存储引擎（TiFlash）的高并发 `INSERT` 或 `UPDATE` 操作？
```
Comment (shichun-0415) [inline faq/tidb-faq.md:99]:
```suggestion
### 1.1.14 TiDB 如何在不影响性能的情况下支持对列存储引擎 (TiFlash) 的高并发 `INSERT` 或 `UPDATE` 操作？
```

File: faq/sql-faq.md (hunk 6 above)
Comment (TomShawn) [reply faq/sql-faq.md:21]:
```suggestion
对于添加索引等 DDL 操作，TiDB 需要进行回填 (backfill) 操作，这个过程需要较长的时间（取决于表的大小）和额外的资源消耗。对在线业务的影响可调节。TiDB 可以通过多线程进行 backfill，资源消耗可通过以下系统变量进行设置：
```

File: faq/manage-cluster-faq.md (hunk 5 above)
Comment (TomShawn) [reply faq/manage-cluster-faq.md:416]: LGTM，谢谢啦

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**